src/lib-rust/src/logging.rs

```diff
@@ -0,0 +1,161 @@
+#![allow(unused_variables)]
+
+use crate::locator::{LocationContext, VelopackLocator, VelopackLocatorConfig};
+use std::path::PathBuf;
+
+#[cfg(not(target_os = "windows"))]
+use std::path::Path;
+
+#[cfg(feature = "file-logging")]
+use simplelog::*;
+#[cfg(feature = "file-logging")]
+use time::format_description::{modifier, Component, FormatItem};
+
+#[cfg(target_os = "linux")]
+fn default_file_linux() -> PathBuf {
+    Path::new("/tmp/velopack.log").to_path_buf()
```

Comment:
I _think_ the rust equivelent here is `stdtemp_dir()` to C#'s `Path.GetTempPath()` useful here?